### PR TITLE
Readds data dirs so pyinstaller picks up non-package .py files

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: ci/pyinstaller/hook-watchmaker.py [hunk 1]
```diff
@@ -6,7 +6,13 @@
 from PyInstaller.utils.hooks import (collect_data_files, collect_dynamic_libs,
                                      collect_submodules, copy_metadata)
 
-datas = []
+datas = [
+    ("src/watchmaker/static", "watchmaker/static"),
```
Comment (eemperor) [inline ci/pyinstaller/hook-watchmaker.py:10]: Minor nit.  Can you match the formatting for dirs?

Comment (lorengordon) [reply ci/pyinstaller/hook-watchmaker.py:10]: This is how `black` formats the file, I don't think we should fight with the tool.

**Comment by lorengordon** [comment]
go codebuild go

File: ci/pyinstaller/hook-watchmaker.py (hunk 1 above)
Comment (eemperor) [reply ci/pyinstaller/hook-watchmaker.py:10]: That's interesting.  Okay, it's good with me.

**Review by eemperor — APPROVED** [review]

**Pull request merged**